loadable_apps/micomapp/Makefile

```diff
@@ -53,21 +53,12 @@
 -include $(TOPDIR)/.config
 -include $(TOPDIR)/Make.defs
 
-ifeq ($(CONFIG_BUILD_PROTECTED),y)
+ifeq ($(CONFIG_APP_BINARY_SEPARATION),y)
 LINKLIBS = $(patsubst $(LIBRARIES_DIR)/%, %, $(USERLIBS))
 LDLIBS = $(patsubst %.a, %, $(patsubst lib%,-l%,$(LINKLIBS)))
-endif
-
-OBJEXT ?= .o
```

Comment:
Generally makefile will have MACROS for everything. Coz changing would be easy later. For example to build on windows DELIM should a '\' backslash whereas for linux it is '/'. If this MACRO is not used. then later to change the build enviroment individual lines should be tracked and changed.

Reply:
Right. But we already use '/' only. And we don't support Windows officially. When we support Windows, we had better modify all of Makefile with the same way. I don't want to use it different way in each file.

Reply:
@sunghan-chang if we dont support windows officially, then why the code is here?
REmvoe all related to windows.  Un-used code should NOT be committed as per coding rule. I am blocking this PR.